--- run.cns	2020-07-07 11:34:16.000000000 +0200
+++ ../run_ref_for_patch.cns	2020-07-07 11:34:07.000000000 +0200
@@ -92,7 +92,7 @@
 {===>} fix_origin_mol1=false;
 {* Is molecule 1 DNA? *}
 {+ choice: true false +}
-{===>} dna_mol1=false;
+{===>} dna_mol1=true;
 {* Is molecule 1 a cyclic peptide? *}
 {+ choice: true false +}
 {===>} cyclicpept_mol1=false;
@@ -2311,13 +2311,13 @@
 {* Number of fully flexible segments for molecule 1                  *}
 {* Note that current max is 5 (edit the run.cns to add more segments *}
 
-{===>} nfle_1=0;
+{===>} nfle_1=1;
 
 {* Fully flexible segments of molecule 1 *}
 {+ table: rows=5 "segment 1" "segment 2" "segment 3" "segment 4" "segment 5" cols=2 "Start residue" "End residue" +}
 
-{===>} start_fle_1_1="";
-{===>} end_fle_1_1="";
+{===>} start_fle_1_1="1";
+{===>} end_fle_1_1="1000";
 {===>} start_fle_1_2="";
 {===>} end_fle_1_2="";
 {===>} start_fle_1_3="";
@@ -3672,7 +3672,7 @@
 {===================== topology and parameter files ======================}
 
 {* topology file for molecule 1 *}
-{===>} prot_top_mol1="protein-allhdg5-4.top";
+{===>} prot_top_mol1="dna-rna-allatom-hj-opls-1.3.top";
 {* topology file for molecule 2 *}
 {===>} prot_top_mol2="protein-allhdg5-4.top";
 {* topology file for molecule 3 *}
@@ -3713,7 +3713,7 @@
 {===>} prot_top_mol20="protein-allhdg5-4.top";
 
 {* linkage file for molecule 1 *}
-{===>} prot_link_mol1="protein-allhdg5-4-noter.link";
+{===>} prot_link_mol1="dna-rna-1.3.link";
 {* linkage file for molecule 2 *}
 {===>} prot_link_mol2="protein-allhdg5-4-noter.link";
 {* linkage file for molecule 3 *}
@@ -3754,7 +3754,7 @@
 {===>} prot_link_mol20="protein-allhdg5-4-noter.link";
 
 {* energy parameter file for molecule 1 *}
-{===>} prot_par_mol1="protein-allhdg5-4.param";
+{===>} prot_par_mol1="dna-rna-allatom-hj-opls-1.3.param";
 {* energy parameter file for molecule 2 *}
 {===>} prot_par_mol2="protein-allhdg5-4.param";
 {* energy parameter file for molecule 3 *}
@@ -4376,14 +4376,14 @@
 {* Setting for the rigid-body (it0) and semi-flexible refiment (it1) *}
 
 {* number of structures for rigid body docking *}
-{===>} structures_0=1000;
+{===>} structures_0=2;
        keepstruct_0=&structures_0;
 {* number of structures for refinement *}
-{===>} structures_1=200;
+{===>} structures_1=2;
        keepstruct_1=&structures_1;
        keepstruct_2=&structures_1;
 {* number of structures to be analysed*}
-{===>} anastruc_1=200;
+{===>} anastruc_1=2;
        anastruc_0=&anastruc_1;
        anastruc_2=&anastruc_1;
 
@@ -4391,7 +4391,7 @@
 
 {* Sample 180 degrees rotated solutions during rigid body EM?   *}
 {+ choice: true false +}
-{===>} rotate180_it0=true;
+{===>} rotate180_it0=false;
 
 {* Sample 180 degrees rotated solutions during semi-flexible SA?*}
 {+ choice: true false +}
@@ -4407,7 +4407,7 @@
 
 {* Randomize starting orientations? *}
 {+ choice: true false +}
-{===>} randorien=true;
+{===>} randorien=false;
 
 {* Expand starting orientations? *}
 {+ choice: true false +}
@@ -4425,14 +4425,14 @@
 
 {* Perform initial rigid body minimisation? *}
 {+ choice: true false +}
-{===>} rigidmini=true;
+{===>} rigidmini=false;
 
 {* Allow translation in rigid body minimisation? *}
 {+ choice: true false +}
-{===>} rigidtrans=true;
+{===>} rigidtrans=false;
 
 {* Number of trials for rigid body minimisation? *}
-{===>} ntrials=5;
+{===>} ntrials=1;
 
 {* initial seed for random number generator *}
 {* change to get different initial velocities *}
@@ -4465,19 +4465,19 @@
 {===>} tadfactor=8;
 
 {* Number of EM steps for translational minimisation? *}
-{===>} emstepstrans=1000;
+{===>} emstepstrans=10;
 
 {* number of MD steps for rigid body high temperature TAD *}
-{===>} initiosteps=500;
+{===>} initiosteps=5;
 
 {* number of MD steps during first rigid body cooling stage *}
-{===>} cool1_steps=500;
+{===>} cool1_steps=5;
 
 {* number of MD steps during second cooling stage with flexible side-chains at interface *}
-{===>} cool2_steps=1000;
+{===>} cool2_steps=10;
 
 {* number of MD steps during third cooling stage with fully flexible interface *}
-{===>} cool3_steps=1000;
+{===>} cool3_steps=10;
 
 
 {======================= Solvated rigid body docking=======================}
@@ -4554,17 +4554,17 @@
 
 {* number of structures for the explicit solvent refinement *}
 {* the n best structures will be refined                    *}
-{===>} waterrefine=200;
+{===>} waterrefine=2;
        structures_2=&waterrefine;
 
 {* number of steps for heating phase (100, 200, 300K)?      *}
-{===>} waterheatsteps=100;
+{===>} waterheatsteps=10;
 
 {* number of steps for 300K sampling phase?                 *}
-{===>} watersteps=1250;
+{===>} watersteps=10;
 
 {* number of steps for cooling phase (300, 200, 100K)?      *}
-{===>} watercoolsteps=500;
+{===>} watercoolsteps=5;
 
 {* write additional PDB files including solvent ?           *}
 {+ choice: true false +}
