--- run.cns	2019-03-12 16:49:37.553879574 +0100
+++ ../run_ref_for_patch.cns	2019-03-12 16:49:30.993433909 +0100
@@ -3047,7 +3047,7 @@
 
 {* Do you want to randomly exclude a fraction of the ambiguous restraints (AIRs)? *}
 {+ choice: true false +}
-{===>} noecv=true;
+{===>} noecv=false;
 
 {* Number of partitions for random exclusion (%excluded=100/number of partitions)? *}
 {===>} ncvpart=2;
@@ -3065,7 +3065,7 @@
 {* Do you want to define center of mass (CM) restraints to enforce contact between the molecules? *}
 {* Note that these are only active during it0 and it1 *}
 {+ choice: true false +}
-{===>} cmrest=false;
+{===>} cmrest=true;
 
 {* Define tight CM restraints? *}
 {+ choice: true false +}
@@ -3314,40 +3314,40 @@
           cols=9 "type" "firstIt" "lastIt" "Kpcs<br>(hot)" "Kpcs<br>(cool1)" "Kpcs<br>(cool2)" "Kpcs<br>(cool3)" "R" "D" +}
 
 {* Number of PCS restraint sets *}
-{===>} numpcs=0;
+{===>} numpcs=3;
 
 {+ choice: "NO" "XPCS" +}
-{===>} pcs_choice_1="NO";
+{===>} pcs_choice_1="XPCS";
 {===>} pcs_firstIt_1=0;
 {===>} pcs_lastIt_1=2;
 {===>} pcs_hot_1=100.0;
 {===>} pcs_cool1_1=100.0;
 {===>} pcs_cool2_1=100.0;
 {===>} pcs_cool3_1=100.0;
-{===>} pcs_r_1=1000;
-{===>} pcs_d_1=10000;
+{===>} pcs_r_1=1252.81466;
+{===>} pcs_d_1=10699.98682;
 
 {+ choice: "NO" "XPCS" +}
-{===>} pcs_choice_2="NO";
+{===>} pcs_choice_2="XPCS";
 {===>} pcs_firstIt_2=0;
 {===>} pcs_lastIt_2=2;
 {===>} pcs_hot_2=100.0;
 {===>} pcs_cool1_2=100.0;
 {===>} pcs_cool2_2=100.0;
 {===>} pcs_cool3_2=100.0;
-{===>} pcs_r_2=1000;
-{===>} pcs_d_2=10000;
+{===>} pcs_r_2=-1130.79587;
+{===>} pcs_d_2=-2705.36877;
 
 {+ choice: "NO" "XPCS" +}
-{===>} pcs_choice_3="NO";
+{===>} pcs_choice_3="XPCS";
 {===>} pcs_firstIt_3=0;
 {===>} pcs_lastIt_3=2;
 {===>} pcs_hot_3=100.0;
 {===>} pcs_cool1_3=100.0;
 {===>} pcs_cool2_3=100.0;
 {===>} pcs_cool3_3=100.0;
-{===>} pcs_r_3=1000;
-{===>} pcs_d_3=10000;
+{===>} pcs_r_3=1491.01656;
+{===>} pcs_d_3=7282.39968;
 
 {+ choice: "NO" "XPCS" +}
 {===>} pcs_choice_4="NO";
@@ -4373,14 +4373,14 @@
 {* Setting for the rigid-body (it0) and semi-flexible refiment (it1) *}
 
 {* number of structures for rigid body docking *}
-{===>} structures_0=1000;
+{===>} structures_0=2;
        keepstruct_0=&structures_0;
 {* number of structures for refinement *}
-{===>} structures_1=200;
+{===>} structures_1=2;
        keepstruct_1=&structures_1;
        keepstruct_2=&structures_1;
 {* number of structures to be analysed*}
-{===>} anastruc_1=200;
+{===>} anastruc_1=2;
        anastruc_0=&anastruc_1;
        anastruc_2=&anastruc_1;
 
@@ -4429,7 +4429,7 @@
 {===>} rigidtrans=true;
 
 {* Number of trials for rigid body minimisation? *}
-{===>} ntrials=5;
+{===>} ntrials=1;
 
 {* initial seed for random number generator *}
 {* change to get different initial velocities *}
@@ -4465,16 +4465,16 @@
 {===>} emstepstrans=1000;
 
 {* number of MD steps for rigid body high temperature TAD *}
-{===>} initiosteps=500;
+{===>} initiosteps=10;
 
 {* number of MD steps during first rigid body cooling stage *}
-{===>} cool1_steps=500;
+{===>} cool1_steps=10;
 
 {* number of MD steps during second cooling stage with flexible side-chains at interface *}
-{===>} cool2_steps=1000;
+{===>} cool2_steps=20;
 
 {* number of MD steps during third cooling stage with fully flexible interface *}
-{===>} cool3_steps=1000;
+{===>} cool3_steps=20;
 
 
 {======================= Solvated rigid body docking=======================}
@@ -4555,13 +4555,13 @@
        structures_2=&waterrefine;
 
 {* number of steps for heating phase (100, 200, 300K)?      *}
-{===>} waterheatsteps=100;
+{===>} waterheatsteps=10;
 
 {* number of steps for 300K sampling phase?                 *}
-{===>} watersteps=1250;
+{===>} watersteps=125;
 
 {* number of steps for cooling phase (300, 200, 100K)?      *}
-{===>} watercoolsteps=500;
+{===>} watercoolsteps=50;
 
 {* write additional PDB files including solvent ?           *}
 {+ choice: true false +}
